wa/framework/workload.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, target, **kwargs):
         self.asset_files = []
         self.deployed_assets = []
 
-    def init_resources(self, context):
+    def init_resources(self, resolver):
```

Comment:
Why does `init_resources` now take a resolver?

Reply:
Because it only needs a resolver, since that is all it uses, it doesn't need a full context (note: in practice, it will still get passed the context, since we're now using that as a resolver, but the name change here is to indicate that the supplied object is only expected to be able to get resources).

Reply:
In which case should this be updated in the subclasses of `Workload` as well to reflect the change?

Reply:
Good point. I have made the change.

Reply:
And also this probably means that the documentation for creating workloads and the workload templates need to be updated as well.

Reply:
Updated.

Reply:
Sorry I meant the create command workload templates and I just realised the resource resolution / deployment documentation will also needs changing.

Reply:
Ah, I didn't realise we had any :).  Will do...

Reply:
All done. Also spotted and fixed a minor inaccuracy elsewhere in resource docs.